main.py

```diff
@@ -20,10 +21,35 @@
 
     run_parallel = True
     manager_runner = XlsxParallelManagerRunner() if run_parallel else XlsxSerialManagerRunner()
-    projects_xlsx = os.path.join(landbosse_input_dir(), 'project_list.xlsx')
 
+    # The file_ops object handles file names for input and output data.
+    file_ops = XlsxFileOperations()
+
+    # project_xlsx is the absolute path of the project_list.xlsx
+    projects_xlsx = os.path.join(file_ops.landbosse_input_dir(), 'project_list.xlsx')
+
+    # final_result aggregates all the results from all the projects.
     final_result = manager_runner.run_from_project_list_xlsx(projects_xlsx)
 
-    with XlsxGenerator('landbosse-output') as xlsx:
-        xlsx.tab_costs_by_module_type_operation(rows=final_result['module_type_operation_list'])
-        xlsx.tab_details(rows=final_result['details_list'])
+    # Switch to either validation or non validation producing code.
+    input_path, output_path, validation_enabled = file_ops.get_input_output_paths_from_argv_or_env()
+
```

Comment:
I made a few changes here to add a flag in the code for validation so I could play with it while I was debugging. Not sure if we need to add it or not.